lib/cbus_elixir_web/controllers/speaker_controller.ex

```diff
@@ -10,8 +12,14 @@ defmodule CbusElixirWeb.SpeakerController do
   end
 
   def new(conn, _params) do
+    today = Timex.today()
+    meets = CbusElixir.Repo.all(from m in "meetings", 
+      where: m.date > type(^today, Ecto.Date),
+      limit: 5,
+      select: {m.id, m.date})
+    meetings = for {k, v} <- meets, do: {Timex.to_date(v) |> Date.to_string, k}
```

Comment:
I need to read up on the idiomatic way to do this in Phoenix, but at first glance it feels like some of this belongs in a helperish kind of function ("give me the next N upcoming meetings" or similar).

Reply:
Also, unless there's a default `ORDER BY` getting smuggled into the query from elsewhere the one above has a bug...

Reply:
Same thing with PageView here; I'd love to see the `get_meetings` function moved out of the controller and into the view.

Reply:
So you are saying to just completely lift it up and plunk it into the `speaker_view.ex` file? Will the controller still pick up the call?